src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/PropertyNamesShouldNotMatchGetMethods.cs

```diff
@@ -81,6 +81,14 @@ private static void AnalyzeSymbol(SymbolAnalysisContext context)
                 var exposedMembers = type.GetMembers(identifier).Where(member => configuredVisibilities.Contains(member.GetResultantVisibility()));
                 foreach (var member in exposedMembers)
                 {
+                    // We only want to report an issue when the user is free to update the member.
+                    // So we can bail out when the property or method (symbol) is override and the other member (member)
+                    // is not local (i.e. symbol and member are declared
+                    if (symbol.IsOverride && member.ContainingType.Equals(type))
```

Comment:
The comment says "is not local" but the check seems to validate it is local?

Reply:
You are absolutely right! Sadly I cannot simply negate the equal condition as we are looping through the different base types... And for the same reason this check is not enough `symbol.IsOverride && (!member.ContainingType.Equals(symbol.ContainingType) || member.IsOverride)` as we end up reporting multiple times for the same member when the member is overriden and a local member is declared. The only "workaround" I can think of is to access the original declaration to do the comparison but that's adding quite a lot of complexity IMO.

I am reverting to the simple `symbol.IsOverride` check and moving it to the top of the class with an updated comment.